cmake/modules/LXQtCompilerSettings.cmake

```diff
@@ -179,16 +179,14 @@ endif()
 
 
 #-----------------------------------------------------------------------------
-# CXX11 and CXX0X requirements
-#-----------------------------------------------------------------------------
-CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
-CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
-if(COMPILER_SUPPORTS_CXX11)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-elseif(COMPILER_SUPPORTS_CXX0X)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+# CXX14 requirements
+#-----------------------------------------------------------------------------
+CHECK_CXX_COMPILER_FLAG("-std=c++14" COMPILER_SUPPORTS_CXX14)
```

Comment:
I would just force setting the CMAKE_CXX_STANDARD(_REQUIRED) ... and that's all. No checking of compiler flags, no error message on our own. Just let everything to cmake.